src/download-header/index.scss

```diff
@@ -0,0 +1,56 @@
+.download-header {
+  background-color: white;
+  border-bottom: 4px solid $blue;
+  margin-bottom: $govuk-gutter;
+  padding: $govuk-gutter-half;
+  position: relative;
+
+  .left {
+    overflow: hidden;
+    padding-right: 1em;
+
+    h2 {
+      font-weight: normal;
+      margin-bottom: 0;
+    }
+  }
+
+  .right {
+    float: right;
+    border-left: 1px solid $mid-grey;
+    padding-left: 1em;
+
+    .download {
+      font-size: 1em;
+    }
+  }
+
+  .download-modal {
+    position: absolute;
+    border-bottom: 4px solid $blue;
+    padding: $govuk-gutter-half;
+    top: 100%;
+    right: 0;
+    background-color: white;
+
+    .close {
+      position: absolute;
+      top: -1em;
+      right: 1em;
+      text-decoration: none;
+    }
+  }
+
+  .download {
+    font-size: 1.2em;
+    margin-left: 0.5em;
+    &:before {
+      @include icon;
```

Comment:
is this a thing outside of projects? If not then we should move it a level up so components doesn't implicitly depend of project

Reply:
ah yeah, I just seen it below - however it will be defined in 2 places now I'm guessing?

Reply:
I've added the mixin here: https://github.com/UKHomeOffice/asl-components/pull/85/files#diff-e30d20921c2abad49a8461ea88b2c052R11

Reply:
fine fine fine, nobody cares about css anyway

Reply:
![](data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAkGBxMTEhUSExIWFRUVGBgaFxgYGBcYFxcaFxYXGBcYFxgYHSggGBslGxcXITEiJSkrLi4uFx8zODMtNygtLisBCgoKDg0OGxAQGy0lICUtLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLf/AABEIAOEA4QMBIgACEQEDEQH/xAAcAAABBQEBAQAAAAAAAAAAAAAFAAIDBAYBBwj/xAA7EAABAwIEAwYGAgIBAgcBAAABAAIRAyEEMUFRBRJhBiJxgZHwEzKhscHh0fEjQlJiggcVFjM0cpIU/8QAGgEAAgMBAQAAAAAAAAAAAAAAAgMAAQQFBv/EACgRAAICAgICAgIBBQEAAAAAAAABAhEDIRIxBEEiMhNRcTNCgZHRBf/aAAwDAQACEQMRAD8Ao1sQZVcvSeU2JXX4o8S8kn7JGVFew7lUo0kRw1NBI0YU2yVrUnNVhtNVMdimsF0iU0lbOhDE5aQx9PqoqtakwXcCR112KB43iby4gWjSYIshVfFjlGefltKxZPLb+p1/H/8AMjHc+y/xDihdYWH/ABGSp0sScwY3CFvxd4nRV61bWf5Wbbds6ajGKqJo28RPLANj0UNTEAGTOWSzxxRsSbDKEqmO0yjJXxK5UHW4kHbw8lFWxnXLT7IAccBkLqCrjzEK/wAbKeRB2ripi+n9KscYJzQpuKJ6ZxC7z6zNtbf2i/GV+S+g5h8Z9FYOMvJJge7oBhsT/SnGKlC4BKejU0eJO5eaxAMe/eiN4Dj9LlAMSCNL/TRYDCYojSR1UrK8GxQ8WXaZ7Bwzjhb3m3E62tbcrS4DtE13zwJ6wvEMJjSIEk+ZWi4bxNzCAQL+d+t1ak0VLEj2qi8OAIMg7KYLDcA4tzHuywgSRkDpYed1s8LW5hP02TU7M7jRYCcFwJwVlCCcuBdhQpnF1KF1TRNnhHwyreHwJ2T6TkQw9Y7rrSkzyOLBFvZDTwMaK22hyiSrDHoHx/i/KOUTOkCw8VmyZKVs6fj+OpSpHMZxZskA5GCRus7jcc1zjGcZZbqviMYJMZk6T781QxNTMkAR781zcmVyPQ4fHjDohxVe8kwqFWuSIFs7/VdMF0nLeclRdXGY0Of9qoxGykOeCNYv6qt8Uk55lT4h5IAMfqc/qqgeRl00+yckKkycV7X0soHPkm2f0XS0k5D+fJRAf0roW5MsOAdAAuqylok3jY/ZRHdWC2cUoq2M52/aY5J7RaD+lCEocOXMzqu0asfVQtak0woWmWvjaaSpaVWDdUXnpCfTcY6aKqDUg7hsRAsitLEW+Y3vCzFCsrjcXkLQlSgOUzdcMxJBHKTEaePVeh9nuJuAAmbCd48F4zheJuMScsui1XBcfUsc4jXc/QoVpkkrPb6FQOEqUBZjgPFuZrSRAP08YWoampmeqOhOXAuqyjkJLsLihLPFqQRHDtVOjTuidFoAldObPMYYlfiOMFNvvzWO4i/vRe+uh80d4zi2gkOk+GQtZZPG14cBK5fkTbZ6TwsSjGwfiKsOJyUWLrtNrmddFPi6oiY2Q/km5tn5ws8d9nQeuijia4y0Gn8qo6p5e/uuvBJ3k+CiJWpIySk2ONU/YeiaXlILunh+VYFsbzKQ1N/OExrV0NyG6hBc0JPScEg5Qg0rq4V0KFHQF2oUubNc57R7soWPDRkct91IBB0zNtFA29pUtIwclC0xNMTEp9OrdRVMyUxhghQlhrD1IAlHuFYpwtJ95LLUDCL4XGG3TZKkh8JHp/AcdUs2dNNdMt7r0zhWMLm3tA1XhnBOKvD2kWANgvSOF8bJeDykWEXm2ZkbIYuiZI/o3gKdKhw1QOaCNVMmiBJJQuKEs8oFBOLo8EePDwLu9Fn+MYxrCYE52/nZa55dHJw+M7Mxxqo02m83+wWZx7+UZzPgr/FKjiS6fTXdAMW5wP8AK5z+UrO/BcIJEP8A/SDYmBOcT4KPiDzJAP8ArHpeR6qKjlPUgZRMSLZpVKMkGZv3joN0dJMrk2ipSdFzf02UT3SZUgaTkE4U9Yn+kwRTYhTsNhKjNI5q+WjliJn8jRdpU8mkGBdDyGfjsosokiRdTjCnlG6vU6IHgpnZXGSB5P0MWD9gwYYm5tZRuwxAlE+S3vVRViYVqZHjQJe1Ootkwp61OEzDZwMzYHbdMsRxpkb2zkPT+E17IUlVkFStmJJMKWVxKpCeSYhPe3NMe6T5fpWVVCfUmyjCUJzFCrJqLs/BW6FRUWqzhxqo0WmaHhmM5SJuJW87M8WBfzE7CenhsvNMI7daHhVbvtE92UmSpmiLtbPdezWKJBBdIkx5k5dFoFiuxZEANuM531t5rahGuhMtMSSSSsEw3EsQQxxGYWI4wye8T1t43W4xIkEFef8AaHmDoEjNFn0hHiu5Gaxx1kdN0FxDS4wdZ8PNHcXVFoIPkg+JzJmNFli9nTfQHPdkRnku1HQAZudPDdOqUZJkTtupjw8yDOW+1809tLsTxb6IaDRmHXdorVOiQ7Pu/nMpnwQ20Z/Tz8lbwuRGaXOWhuOG6Y+lSuSrLcNOWimw9HplpurdOkTdosszk7NscaSBtKjzXMACc0w4cmNDkrzsKWuB01TuS9xCqwuCBnJBny8VA+nZFMXS1geSoRJRxlsXOBQLZMRko6FFX3UJuuCndO56M34neytUpAmUxxE8qvmlKgrYe8q4zRJY2gfVZHgq7iEUrMEBDKzYKbF2Zska2MldauJw2hEKHjorNEBQ8uuysYd26tloI4UI1w1xa6wv5XQOk7oi/Dn94JMjRA9a7D4491oEDXzyBH5XojDIlee9hKjCW8umfjAleiNVx6FT7OJJ0JIwDB1TZYvtF3uYnux/S2zhaVhO0TgH8smTeUWf6mXxfuZbEUYk2tl1QLGuGmhy80ax7db/AMoRVggvsIzGpGyyRezrNaKpeDkL2+uSnwdV3Ne/+t/fj6qjWkER72VhzwQCDByPU7lMa0DF7LbqRsDnt00UlAQbX9yoqDgTzDqR5K3Rpd63kkS1o0wV7DNGl6nLoFfo4flAvITcBSMCURxlIDlAPis9moDY8NJgX18VSa03WlZgmmJb+1VxlFsWzVpkQFqUGnNRU8IDIGyK18IYkDRD6LCHRuiTLdFSrhrRsqzaImEZxFAwVVoYMAlzkSYuSVlF+FOihNCM7q/Xx1NuZAKGYrijDa/kmRjJiJTgu2MqU+iF41kFEaePaf3ZVca2QnwtPZly8ZLRQDV0DVc5SpGT4JplJWdM7+EKQMuoqRKmZKhZaw7skcwGf7yQOjlCJ4R10uSGw7PYP/Ddgl2cj09V6WwWXkf/AIe42HtbNtfDZeuMdKuHRWT7HV1JJGLPEuGY2o08jw+m4HvNMx5bjwQztBU/yWGmetwt3gX08bTJfT5XCxGVx/s05i6xva7DcroLrjXU9TCvO/jRl8KNTu7Rl8TMIDi2G59+KNV3RafcW8ELxI2WOOmdeStA2kLkT1HUj8p9YC9rxJ8ypalOBfMZR16KnMnWQU9bFPWibA1C0kewjvC3Aube5P2QNrBMzYovwJh57afRJzJNWaPHbTo3GBAA5zAAlUMfxpjLAAxkfD8ohTwlWo0MbAadTZPfwLD0W81UNc71nwAN1niv2apSM5U7QOnutuRGqdgsbVJ79O2hU/EeK02Bxo4aWtaXF0hohpAMbxIyQ7h/aF7ocGgA3vcQLG+iNw1dARmrq9mnpVA4QQckLxVPlqtMW5gEa4RjG1nNBbyk5ag2vBCvcb4WIDoug6G9GW7R1AyI9UDxPeyKJ8WufBZ3HYgjK338AmQV9CsjpbLFLgoeZLgPNSu7OU235wT4oTVpVHs7ocScoMcu8/8AKfwrGGwNQX5XNtlzE33un7r7GTTlXEnqcKAFhCHVsLHkirC9p72R9PRNqidM0Cm0MeNNaAFSjfwCjqATPvJX67COirOZIyg7rRFmKcRrBaRbf8KwG6qJjbESpWWtsiBJaIRXBiIQ/DiSjPDacuEoJMZjRvuwOHl5cNxygbz9gJXsVAWWF7DcOhrS0ZE8x9Ijyst+0KRBm9nIST1xGBZ5Nw/FOpXpkOcQLZz+lV7a4D/fV19oysIzU1FwpfLygBotOs+cJ3GsXz02jlHLG5MGMim542jn+HPjKrPOeKYYkyPVCKtMxAz16XR/H0XZBp9LIbiBt+1zbaO9HaBNWYI1UbaUEzt+FYxLNVDy7z08fynJ6Aa2QXAjWb+S0fZSnz1J8PqgTwBfb65LQdhz/mj/AJIMv1G4dTPWMBhWsYN/BBqnAvjVS6oSGizW5TvIFwtJRd3QBbrnH7Q/EOc0922usnxWdGqKBvFuzVH4QYflGQB+XeNll6/B6VMRTaPL9rW4jFF3zCU2ng2uNxbO26JybJGKjtg3s9w8gCBDeYGNiNR9VpeIwaZ6KNz4tEBRVXmIQjKvZgOMHvnZAsRhNUe400/EPihxokhFF0BOKG8NqlliJRGpiA4Xt5H+FSpUosieFarspIE1M8r9U0MGojqEZxdAZoeWQpZGgLxShAndBTndabirO7KAso3WrE/ic7PH5DALKZosl8A3spKVOdf5TLENFjCsWz7O0WkAnUgExlKy2Bw5OS2fAKYbBnWL5Tt90uTGxVI9l7NMApNDQA3SLo2EB7K04pAQR0KPtCOIiRxJdhcRFWeF4Vr306dRxjmY0xvIuilMuqM5Tqc9kE7FVWVaTWF4Lqdg0EmGknlzWna91OzSIOdlqu0caMeMgHxnhbm0wfiWBA281isTnFvH8r0vjWLpvpuZEk7XiF5xi8OQ4gjLdcvPXPR6Dw53GmUH0wc1RqMiY93+iJVaefp5qk5ligizXJWUqrD4zotL2CZOJYDtfyQRtIxMfwtP2GpRiGdQryT+NB44U7PUgYXMYwlsgLoKuVMU1tKIk7pURttdGbqNAN1ylxINdytAJJgITxzicTdVuy7nvqfEIJaLKDaVbNe6uD3XAc3QQFBVYCFbLqQMkwfFce6lyuN5i2ylAckjA9oKMOmFn6r3TAyWp4zWBkRN81nn4pgNrnp/KtIjaKdUOYZRDBY0FRVjzC6GEmm6dPsiSsHlRp3ulVXsUeErcwzUhVBN6KGLpy0hBqLAXI9VNj6LL4mZMJuJXox53xaZbxeJaCGtvvGnRKiyT196oVSsjHD2zpKfx4oyPJ+SVsKYShDrGPC62PZjAh72i5M3/mEC4JTPOAQD0OXgvRezlGKzSGgGSSLiItaLIBjdI9B4Zh+RgBvYXOeSvwoqWSlTUZmcSXUlZR8z9isefjimS1jYJPM3le46fQLccaxvw2tA+Z5gfleXUMSaZdUbUFQ5y652gR4LRYfiLq7WPuYEeCNZPjRgljXKzQ/GaQDF7zlCzvFao5oEAxvpKJGqCzX1g/pDMa0RyxlF7Em1rrnTUpStm3DJQqgTiGWsfJUKmeUIpTktuPZUOIE2IvZD9WdKE+S0Q8PxYZIcJac9x1RzgFdgr0yHCC60dcln3UjGUKfBv5Hc2xEKSSZoxzdUexB6ocSrQ07JYHEhzGnf8oL2xxJZTMJafobEy+Lf8WsGDU+g1WzwOIZQYGti38LEcJBYOY/M6JdtKLUca11nOEF2ci4n8pl0C5WG+K8f5SBa4vbLqsziuPOs8OIEwRofLdNxXfMgzY5baITWwz/l5bbx9VS2V0W8c8vlhJjfob+/FURghpbZXKTeVkEmdM5H6UFQS03+XLzv90SBkiqXuaeU+91UqYi5VriGJ5WhzhlmOsaIMazn3DI1ToRvZnnka0i9SxpZBGU3Hij9GvzCd1nKdMkAxnYo3TcGU52CGdDMUnTsjfU7hjcoDVz8UR54afeaoVAmY1QjO7K9IZ29UbwNOYj30VWjQBHVans9wWo8d2nzCfm/4+H8pkmIhGmF+CNfm5uTr7g6wvUOAYR5c1/NLZ+gtA3ndZ/sr2afUcX1JawHzfvHSdV6JhMOGNDRkEMUSciw1PXAupiFMSSSSso+SMMxrXOe4w1vLac4ziM7Stbwr4PKTSdLdjudp0WEaHlpcTfJoi3hsCjvCezteryVW1GBoIsQeYEHvAgKGRpBvitQsNMZAuvtFhdMxWFLXEk2sZi9yJyzEJnbAlppA6NJPqLq61gfSY/mAAaAXX8PI/ylZOi4dgfHPDHB3k0TOxJnTPXdR1ny2eX7K9j6TSItIGeR6GdbrNHEvYTPfPUnzjdJjDmjVDK4MJBupKa9nezUFLiDbBwI5t/1krRIz9PBLcWuzo4siktGn4BjZZ8PbLwzlEON0hWZOkfVZPC1uUSM/PJEv/MC4covOdoAOyVRrjIHYzugU9zJ8NPoFyt2bbXH+MnqNuoSxTXEkjP+bfZaXsuIjdHF0SlRHwGgKTfhVmX5SOeLGDInqj1CvhnGm8R8jhBEd4AC4OuausexwIeLIVj6DBYAEK2l2VxT1tAytjMO2o+SwSBFxnJ/SzHE8QP8jWNJkgj/AI6I9WwzJkNG6oYl7RkFEi/xL9sy2Kwj6juZ1ha2ggKSphQ1sBX675kj+lTxBJbGqZyfQtxirpCotBby5ERHiBBBUOLdMAZH8LlB8ROkwfwVGa3fk6o1HYhy1RFijkNk7CUS5zQBqJ+ygebwtHwHh8lp2unRXoyzl7NX2W7O06zgz4QLs55uWG63/S9U4F2cp4emGATnPmZhea8IxjqFVtRubTluNQvX8JiG1GNe0yHCQrcaYpStE1NoFoUgTWp4UIPC6uBdVoESSSSsh8h/EpQOS+8znuBotBwXtRyH/IWuaT3jEOmwBJ1jJY1z+9aQNBFr5eKscMw7nGp3iHhpLQMo/wBhGv6VtGbiqs0/arira4pkNLXQ6bggXt56pnDeIuGGIJycRG45QMuhCFYED4obVJ5JzmYMQLlaI8MptB5O7rmSJnrb+wkZc0YOmgseCWRXFoDY/iMBudxnlnJy8CPRD6/EQNc89oWgdhQS5rhPdMWAGQktPn9Fm8XwIgFzSCCc9IsPzPgqx5IPsa8LRVq4sugkT1+ysYTicAtJyyJ1GyFOlpLTpkCnVmSAffuE9xTRINwZrMFiCbbj6eyr1HEBndEwbk+N7BZXh+OFpJAm9s9h72RSm6Xe/eSyTx0zpY81o0tIhtzr7t6+iIcMqf7A+X5QoO7jfCY/J9RARDhzYE5e/RKcR8cl9hjFYmx5ZnfQfyhVau82l2l4RHAyD3gDf75ef8K1xDlDTbr/AH4oUNv0ZSrWMed/BUXtg8rjEjPrmPJFMZTDgYtcITUpHxj7fyiRTTKVV3KZBuM/eyrPrzpF1dxTQdcwq1BiZFCZuiKrTm4yKG1XHmGwKNVqXKQBqbed1Tq4UueHDwPitETFNjOHYYvMa3geK9A4fgwxo3zlCuz/AA/lHNHzH6LROTor2ZckvQwtW17E8cY1nwarw2/cJMC+k6X+6xkKOv8AK7wUaBjKme3ApzV5d2S7WPokU6pLqXqWeHTovTqFZrgHNIIIkEagoGhydk4Tk0JyiKYkkklZR8YOjQlS4fEuYZbBnpoVFRwTnfKC63nPRSvwrmuaHyy4neNfor5LoS4OjT8GZQqMDpL6lubSDlki9bEAODeQmRaLgDKUM4VSpNH+PJ2p1gja4Vx9Uhp5jJJMAQLdVycrcpf9OhjSjHRHWEAhzy6Z00vY7qlSe0N7tpFgOgA16Ks+o9j3Emds4ibn0KtNu6bcrZv/AFomcaF8rBfE+Gse0PHdOZy1EnPPTwQt9KGNJuSSP39le4hi6j3OayC24Hd0Jgq5ieHt5GhpN4z0PTyWuEuKSl7ETXLa9GaewB/KTnHQefktbhcKJDmkGwEC8wUDxGCLnBpMGIPkYH0hHcIwNYwM/wBBdw1y11UyvSoPFJoNUyHPDQPmE9LRI8gVYqO+HJmGxDY3Pv6KXhvC6jwK7W92nILsp5rQPQeijeA6mQRdptPWMktdWaVImoY0tBcfIXyhV6nES5pE6ff2FRxTzBJJmTHmCECfjjbPP7WQOF9GmGVLsO0MWASCclXq1dR6eKAuxsknf3Cndji5thBafUeyrWNhPMgjVHML7Ej+feyhNK4IzPTUe/olh3EiJMxH0v8An6K9Tp6/9QP009UxKjNOdkFdg5GWkz9v3HqrfC8BJkhTMw3MW9P7KNUaUCE/HEyZJ+iSjTDQOi6SuhdITjOMBVbH1cmjzU9WpyNLj5D7BCmy4k7qqLRYpuWw7I9ozQIpvM0ifNpOo6dFkmUirFJpCFoNM9yovDgCDINwRkVKF5l2X7SOoEU3kmkfVvUdOi9Hw2Ia9oc0hzTkQhqgrsmhJNlJQh8z4fBNpN7hJtIaTbyhR18Sx5io3vRkRfXIqTnDzaW8pictMr5ruKIMWkWnL3dcu92+zqUuNLoi+DAlgOsD7WVbGVsmZlomTkZvvmrTJa3nD5BPpO6q4+jJ+ILwIcNxKOHezLkikm1/ooYhxjczbYTp5hScOc+nzGYGfjOX3iE6nTBb3AYkZxaPZXaeGJe4CYfk7OI39D6rT8eNMx/Llov1a4gOjMSIznMj3sq1OtLSTMu06j9JmJraNMcvu3vVVRXAPPcuOhyQRhoNz2O4hTkh5NhBkDw+tloOwfAX4x/LBbSYe+7xyaP+pU8BhvjuZTdIDngFwGQJzk5WXtnZzDUaeHa2gwsYJF8ydXHcndOxrkqfoF/F2idvD2NpCixvKwCAB7ush2k4KWN523mBpbbyW1qPvCp8Toc9Nw3FvEJs43ELHKpHjeMokNh03ceuqCYuhGYuTl03O0rf4rhoJmY2tIWY4rw9zJADnDff1WRTOk8TMjUbfwzV/CUIuRmimB7PPfdxjxRhnBABd0+ACN5V0DHBJgnCsgi2Q8rxKJ0afxNIEn+/wpW8HLzF41RrC8O5QAjxx5bM3kS/H8fZXwmF5QArYpq0yinigtSRhbsrNprhYrnwCqXGK3wqdvncYaOv6CugbBeMJqP5Rk23nqieC4YYyV3g3BoaCc9UcZRA99FAkA3YGNFC7D+/NH61MX8/wh+JpZ+apogLLIRrs9x5+HdGbDm38jqhz2/lQlqFoJM9H/8AVdDd3/5/aS82griGg+RkqWf/AH/gqvjP/c/7R9wkkuT7Oq+jlL5He91Xo5O8D9ykknLt/wAmXJ1/ghwXynxCv8J+ZySSrL7Ax/ZA1+Q97Ku7/T3/ALJJLVHozPtmw4d8o8vwvYsF8jfAJJJXh9yNPk/2/wAEGJ+YeafifkSSW8ymLxGviUFx/wAySS5LO/D0NpZDxKu6DwH2SSVDF0EOH/L6qYpJLq4/ojz3kf1ZfyNCnYkkmmcfqPH8LPdov/kYb/7H7hJJWRdmywnyBdb79EklQSIq3v0VHE/ykkqZZTqfyqz/AH6JJIWQjSSSQhn/2Q==)

Reply:
Yes. https://github.com/UKHomeOffice/asl-components/pull/85/files#diff-e30d20921c2abad49a8461ea88b2c052R11-R19